docs/bagging.md

```diff
@@ -17,10 +17,10 @@ Baggers do some quality assurance on the dataset to make sure the content is cor
 - The organizers of the event (in-person or remote) will send you an invite to the [Archivers app](http://www.archivers.space/), which helps us coordinate all the data archiving work we do.
   	- Click the invite link, and choose a user name and a password.
 - Create an account on the DataRefuge Slack using this [slack-in](https://rauchg-slackin-qonsfhhvxs.now.sh/) (or use the Slack team recommended by your event organizers). This is where people share expertise and answer each other's questions.   
-- Get set up with Python and the Python script to make a bag at the command line: [https://github.com/LibraryOfCongress/bagit-python](https://github.com/LibraryOfCongress/bagit-python)
+- Get set up with Python and the [`bagit-python`](https://github.com/LibraryOfCongress/bagit-python) script to make a bag at the command line
 - If you need any assistance:
      - Talk to your DataRescue guide if you are at an in-person event.
-     - Or post questions on Slack in the #Baggers channel (or other channel recommended by your event organizers).
+     - Or post questions on Slack in the `#baggers` channel (or other channel recommended by your event organizers).
```

Comment:
Hmmm, good question. I was just updating to the DataRefuge Slack name. I think we are okay with that, but maybe we specify "DataRefuge `#baggers` channel"?